static/js/app/controllers.js

```diff
@@ -11,61 +11,62 @@
             .then(renderStatus);
     });
 
-    app.controller('DashboardCtrl', function($scope, $cloudfoundry) {
+    app.controller('DashboardCtrl', function($scope, $cloudfoundry, $location) {
         // Render the orgs on the page
         var renderOrgs = function(orgs) {
             $scope.orgs = orgs;
-        };
-        // Render a specific org to the page
-        var renderOrgDetails = function(data) {
-            $scope.orgDropDownName = data.org_name;
-            var spaces = data.resources;
-            if (spaces.length > 0) {
-                $scope.activeSpaces = spaces;
-            } else {
-                $scope.activeSpaces = null;
-            }
+            $cloudfoundry.setOrgsData(orgs);
         };
         // Get data for a specific org
         $scope.showOrg = function(org) {
-            $scope.activeOrg = org;
-            $scope.visibleTab = "organizations";
-            $scope.activeApps = null;
-            $cloudfoundry.getOrgSpaceDetails(org)
-                .then(renderOrgDetails);
-        };
-        // Clear the dashboard
-        $scope.clearDashboard = function() {
-            $scope.visibleTab = null;
-            $scope.activeOrg = null;
-            $scope.orgDropDownName = null;
+            $location.path('/dashboard/org/' + org.metadata.guid);
         };
         // Get Orgs or return to login page
-        $cloudfoundry.getOrgs()
-            .then(renderOrgs);
-        // Catch the active space from SpaceController
-        $scope.$on('emitActiveSpace', function(event, apps) {
-            $scope.activeApps = apps;
-        });
+        $cloudfoundry.getOrgsData(renderOrgs);
     });
 
-    app.controller('SpaceController', function($scope, $cloudfoundry) {
-        // Set the current active spaces 
-        $scope.setActiveSpace = function() {
-                $scope.$emit('emitActiveSpace', $scope.apps);
-                $scope.activeSpaces.forEach(function(space) {
-                    if($scope.space.metadata.guid == space.metadata.guid)
-                        space.selected = true;
-                    else
-                        space.selected = false;
-                });
+    app.controller('OrgCtrl', function($scope, $cloudfoundry, $location, $routeParams) {
+        // Display Space
+        $scope.showSpace = function(space) {
+            $location.path($location.path() + '/spaces/' + space.guid)
+        };
+        // Render the org information on the page
+        var renderOrg = function(orgData) {
+            if (orgData['code'] == 30003) {
+                $scope.activeOrg = "404";
```

Comment:
guessing it has to do with https://github.com/18F/cf-deck/pull/30/files#diff-60bb1f6b95d957acd2a186fed9e2db20R3